package.json

```diff
@@ -16,7 +16,7 @@
     "@angular/compiler": "2.0.0-rc.5",
     "@angular/compiler-cli": "0.6.0",
     "@angular/core": "2.0.0-rc.5",
-    "@angular/forms": "2.0.0-rc.6",
+    "@angular/forms": "2.0.0-rc.5",
```

Comment:
@catherinemarston 

could you explain why rc5 would be better than rc6 please?